app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Consider using more descriptive variable names for `parts` to improve code readability, as suggested in the checklist. For example, `command_parts` could be a more descriptive name.